praw/__init__.py

```diff
@@ -972,10 +972,16 @@ def get_rising(self, *args, **kwargs):
         return self.get_content(self.config['rising'], *args, **kwargs)
 
     @decorators.restrict_access(scope='read')
-    def get_sticky(self, subreddit=None):
-        """Return a Submission object for the sticky of the subreddit."""
-        return objects.Submission.from_json(self.request_json(
-            self.config['sticky'] % six.text_type(subreddit)))
+    def get_sticky(self, subreddit, bottom=False):
```

Comment:
Should bottom default to `True` here, so calling `get_sticky` without the arguments returns one that was set via `set` without the argument?